notes/design_principles.md

```diff
@@ -0,0 +1,101 @@
+
+## Design Principles for Data Modeling in HyBox
+
+### Use discovery and search requirements as a pragmatic means to determine the needed extent of the modeling efforts
```

Comment:
Defer to business for decision, but I agree that's a rabbit hole of the deeeeepest proportions. :black_circle: 

Like you, I would prefer to focus on end user value, rather than trying to bake the ocean into a salt cake.  If there are real use cases for additional metadata specificity, lets get them in as feedback and respond then, rather than over-engineering from day zero.